lemminx-liberty/src/test/java/io/openliberty/LibertyCompletionTest.java

```diff
@@ -76,10 +76,28 @@ public void testFeatureCompletionItem() throws BadLocationException {
                 CompletionItem microProfileCompletion = c("microProfile-2.2", "microProfile-2.2");
 
                 // would be 269 if mpConfig-1.4 was not already specified
-                final int TOTAL_ITEMS = 268; // total number of available completion items
+                final int TOTAL_ITEMS = 337; // total number of available completion items
 
                 XMLAssert.testCompletionFor(serverXML, null, serverXMLURI, TOTAL_ITEMS, jaxrsCompletion, websocket,
                                 microProfileCompletion);
         }
 
+        // Tests the
+        // availability of feature completion
+        @Test
+        public void testPlatformCompletionItem() throws BadLocationException {
+                String serverXML = String.join(newLine, //
+                        "<server description=\"Sample Liberty server\">", //
+                        "       <featureManager>", //
+                        "               <platform>|</platform>", //
+                        "               <feature>mpConfig-1.4</feature>", //
+                        "       </featureManager>", //
+                        "</server>" //
+                );
+
+                final int TOTAL_ITEMS = 23; // total number of available completion items
```

Comment:
This number seems high to me. Can you show me what the full list of platforms are?

Reply:
Yes there are 3 javaee, 3 jakartaee and 15 microprofile items